io.github.agam778.MS-Office-Electron.yml

```diff
@@ -0,0 +1,59 @@
+app-id: io.github.agam778.MS-Office-Electron
+branch: stable
+runtime: org.freedesktop.Platform
+runtime-version: "21.08"
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: "21.08"
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node18
+command: start-ms-office-electron
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --share=network
+  - --device=dri
+  - --filesystem=xdg-download
+  - --talk-name=com.canonical.AppMenu.Registrar
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.kde.StatusNotifierWatcher
+build-options:
+  append-path: /usr/lib/sdk/node18/bin
+  cflags: -O2 -g
+  cxxflags: -O2 -g
+  env:
+    NPM_CONFIG_LOGLEVEL: info
+    npm_config_nodedir: /run/build/ms-office-electron/flatpak-node/cache/node-gyp/18.2.4/
+modules:
+  - name: ms-office-electron
+    buildsystem: simple
+    build-options:
+      env:
+        XDG_CACHE_HOME: /run/build/ms-office-electron/flatpak-node/cache
+    sources:
+      - generated-sources.json
+      - type: git
+        url: https://github.com/agam778/MS-Office-Electron/
+        tag: v0.7.2
+        dest: main
+      - type: file
+        path: io.github.agam778.MS-Office-Electron.metainfo.xml
+      - type: file
+        path: io.github.agam778.MS-Office-Electron.desktop
+      - type: file
+        path: io.github.agam778.MS-Office-Electron.png
```

Comment:
there are icons in the source git repository so you don't need to include it here.

Reply:
So, the .metainfo.xml file, .png file and .desktop file should be in upstream?

Reply:
ideally, at least in a future release. 

There is already an icon though.

Reply:
Hmm, So I'll just push these files in the repository for now and then put them in sources.

Reply:
Updated, Please check.